client/src/app/pages/migration-waves/migration-waves.tsx

```diff
@@ -244,8 +244,13 @@ export const MigrationWaves: React.FC = () => {
     return statuses as TicketStatus[];
   };
 
-  const aggregateTicketStatus = (val: TicketStatus) => {
+  const aggregateTicketStatus = (val: TicketStatus, startDate: string) => {
     const status = ticketStatusToAggreaate.get(val);
+    if (status === "Issues Created") {
+      const now = new Date();
```

Comment:
Lets try and use Dayjs here to be consistent with what was done in the wave create modal. Thx.